docs/common/StringResource.md

```diff
@@ -0,0 +1,35 @@
+# Localized strings
+
+All strings shown to the user must be localized. The `StringResource` service provides an easy way to access those strings.
+
+## Usage
+
+### Creating a localized string
+
+The project you are working in should have a directory under `Strings\en-us`, containing a file called `Resources.resw`.
+If it does not, see [Writing a tool](../tools.md#writing-a-tool).
+For information about adding strings to this file, refer to 
+[Store strings in a resources file](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/mrtcore/localize-strings#store-strings-in-a-resources-file).
+Dev Home will take care of getting localized versions of these strings.
+
+### Referencing your localized string from XAML
+
+To learn how to reference your string from XAML, see 
+[Refer to a string resource identifier from XAML](https://learn.microsoft.com/en-us/windows/apps/windows-app-sdk/mrtcore/localize-strings#refer-to-a-string-resource-identifier-from-xaml).
+
+### Referencing your localized string from code
+
+While it is possible to reference your strings from code using the method in the article linked to on this page, for consistency you should use the `StringResource` service.
+You can obtain a reference to the StringResource service with the following code:
+```cs
+var stringResource = new StringResource("<your_project_name>.pri", "<your_project_name>/Resources");
```

Comment:
We've been using `IHost` to get the string resource object.  Will it still work after this change?

Reply:
NVM.  I see it still works.  Just didn't read far enough.

Reply:
Whichever way of getting the service each place was already using, be it new-ing up or getting the service from the Host, I left them alone, so both ways still work. I guess since instances of new-ing up were what needed updating in this PR, my brain went there when I wrote this. I'm not sure how to include that way of doing it without including a bunch of DI information that doesn't really belong in this document.

Reply:
For setup flow, it should continue to work as the service itself was updated, but the users remain unchanged.